pkgs/applications/networking/pcloud/default.nix

```
@@ -0,0 +1,110 @@
+# Even though pCloud Drive is redistributed as a plug-n-play AppImage, it
+# requires a little bit more love because of the way Nix launches those types
+# of applications.
+#
+# What Nix does, simplifying a bit, is that it extracts an AppImage and starts
+# it via buildFHSUserEnv - this is totally fine for majority of apps, but makes
+# it by-design *impossible* to launch SUID wrappers [^1]; in case of pCloud,
+# it's fusermount.
+# (so pCloud starts, but silently fails to mount the FUSE drive.)
+#
+# To overcome this issue, we're manually extracting the AppImage and then treat
+# it as if it was a regular, good-ol' application requiring some standard path
+# fixes.
+#
+# ^1 https://github.com/NixOS/nixpkgs/issues/69338
+
+{
+  # Build dependencies
+  appimageTools, autoPatchelfHook, fetchzip, stdenv,
+
+  # Runtime dependencies;
+  # A few additional ones (e.g. Node) are already shipped together with the
+  # AppImage, so we don't have to duplicate them here.
+  alsaLib, dbus-glib, fuse, gnome3, libdbusmenu-gtk2, libudev, nss
```

```suggestion
  alsaLib, dbus-glib, fuse, gnome3, libdbusmenu-gtk2, udev, nss
```

`libudev` is an alias to `udev`, which our CI doesn't allow. (Don't forget to change in `buildInputs`, as well.)

---

Thanks! I was scratching my head thinking what's going on; so generally all aliases are a no-go?

---

Yep.

For future reference, useful information on the failure can typically be found in the details link -- if you click the red X on your earlier commit, there's a details link that sends you to a gist, showing the failure log. This would have at least alerted you to something strange going on with libudev.